openstack/db/v1/configurations/requests.go

```diff
@@ -165,3 +165,17 @@ func GetGlobalParam(client *gophercloud.ServiceClient, versionID, paramID string
 	_, r.Err = client.Get(getGlobalParamURL(client, versionID, paramID), &r.Body, nil)
 	return
 }
+
+// AttachConfigGroup will attach configuration group to the instance
+func AttachConfigGroup(client *gophercloud.ServiceClient, id string, configID string) (r commonResult) {
```

Comment:
`id` seems ambiguous here. From what I can tell `id` is the instance ID? If so, maybe rename `id` to `instanceID`.

Reply:
makes sense, thank you